src/partials/gallery.html

```diff
@@ -0,0 +1,6 @@
+<section class="section-gallery">
```

Comment:
Zmień proszę nazwę klasy na `section__gallery`.
Tak jak zrobiłaś też jest OK, to jest kwestia tego żebyśmy mieli spójność w nazwach.